static/js/components/StudioList.tsx

```diff
@@ -54,7 +54,7 @@ export function StudioListItem(props: ListItemProps): JSX.Element {
   )
 
   const closeMenu = useCallback(
-    (e: React.MouseEvent) => {
+    (e: MouseEvent) => {
```

Comment:
When I installed CKEditorInspector, some packages got minor/patch bumps (see yarn.lock). In particular, `@types/react-outside-click-handler` got bumped from v1.3.0 to v1.3.1, which changed the event type on these handlers from `React.MouseEvent` to `MouseEvent`. (The old type was incorrect. It has always been a MouseEvent. Nothing in the runtime code changes).